.clang-format

```diff
@@ -68,14 +68,16 @@ ForEachMacros:
   - TAILQ_FOREACH
   - WT_CELL_FOREACH
   - WT_CELL_FOREACH_ADDR 
+  - WT_CELL_FOREACH_FIX
+  - WT_CELL_FOREACH_FIX_VRFY
   - WT_CELL_FOREACH_KV
   - WT_CELL_FOREACH_VRFY
   - WT_CKPT_FOREACH
   - WT_CKPT_FOREACH_NAME_OR_ORDER
+  - WT_COL_FIX_FOREACH_BITS
```

Comment:
Only a couple things changed, but a lot were added, and now they're at least all the same. My inclination would be to leave well enough alone for now.

WT_COL_UPDATE_SINGLE specifically is bad, I agree, but it didn't occur to me to change it. Could do just that (e.g. to WT_COL_FIX_UPDATE) but it becomes a bit of a can of worms, because the VLCS usages are split half and half between WT_COL_UPDATE and WT_COL_UPDATE_SLOT and the latter's shared. Of course, that sharing is one line in btmem.h and could be explained there.

There are probably more WT_COL_* things besides WT_COL_UPDATE that should be WT_COL_VAR_*; this wasn't on the radar before and I didn't go looking this time around. Happy to do a massedit later if you think it will improve things, but it probably shouldn't be part of this change...

Reply:
Agreed; I'm OK with the state of the world, it's better than it was.